Dockerfile

```diff
@@ -37,7 +37,7 @@ ENV \
   DB_PASSWORD="" \
   DB_PORT="5432" \
   DB_USER="" \
-  ETHEREUM_PRIVATE_KEY="7920ca01d3d1ac463dfd55b5ddfdcbb64ae31830f31be045ce2d51a305516a37" \
+  ETHEREUM_PRIVATE_KEY="edc63d0e14b29aaa26c7585e962f93abb59bd7d8b01b585e073dc03d052a000b" \
```

Comment:
Just to confirm, this is a dummy key, right? We should embed something in this to make it clear that it's a phony key, just to help avoid raising false positives in security audits, automated or otherwise.

Reply:
Yep, a dummy key, but it corresponds to an addresses that's automatically funded in the tests via flags passed to ganache-cli. Not sure how I could make this clearer.